cmd/soroban-cli/src/tx/builder/muxed_account.rs

```diff
@@ -0,0 +1,70 @@
+use soroban_sdk::xdr::MuxedAccountMed25519;
+
+use crate::xdr::{self, Uint256};
+
+pub struct MuxedAccount(pub xdr::MuxedAccount);
```

Comment:
I think we can eliminate this type, and most of the conversions between the different things by using the xdr::MuxedAccount after this PR merges:
- https://github.com/stellar/rs-stellar-xdr/pull/396

Reply:
This still requires a FromStr implementation.

Reply:
`xdr::MuxedAccount` already has a `FromStr` impl for strkeys:
https://github.com/stellar/rs-stellar-xdr/blob/67be5955a15f1d3a4df83fe86e6ae107f687141b/src/curr/str.rs#L90-L100